.github/workflows/image.yml

```diff
@@ -0,0 +1,37 @@
+name: image
+
+on:
+  schedule:
+    - cron: '30 2 * * *'
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Load osticket version env vars
+        run: 'cat version-osticket.env >> $GITHUB_ENV'
+      - name: Generate image tags
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: devinsolutions/osticket
+          tags: |
+            type=semver,pattern={{major}},value=${{ env.OSTICKET_VERSION }}
+            type=semver,pattern={{major}}.{{minor}},value=${{ env.OSTICKET_VERSION }}
+            type=semver,pattern={{version}},value=${{ env.OSTICKET_VERSION }}
```

Comment:
Can we perhaps specify `latest` explicitly? It'd be nice to not forget about it if a version-specific branch is created.

Reply:
You would just need to use the [`flavor`](https://github.com/docker/metadata-action#flavor-input) input here and define `latest=true`, This configuration would then tag (given osTicket version of 1.15.4):

`:1`
`:1.15`
`:1.15.4`
`:latest`

I might not be 100% on that , you may wish to try the different combinations on a fork first.